ImaginaryTests-iOS/ImageView+ImaginaryTests.swift

```diff
@@ -0,0 +1,49 @@
+import XCTest
+import Cache
+@testable import Imaginary
+
+private final class ImageView_ImaginaryTests: XCTestCase {
```

Comment:
I see you like underscores.

Reply:
Did you know I'm also a fan of http://underscorejs.org/ 😉